docs/features/A01-wake-window-schedule-generator.md

```
@@ -0,0 +1,66 @@
+docKey: wake-window-schedule-generator
+category: Scheduling & Prediction
+priority: P0
+status: Built
+tags: [core, scheduling, wake-windows, no-account, mvp]
+---
+
+# Wake-Window Schedule Generator
+
+**Competitor verdict:** Table-stakes — every serious scheduler predicts nap timing; our wedge is doing it free, in the browser, with no account and transparent, cited math.
+
+## Problem / user need
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)